package.json

```diff
@@ -30,13 +30,12 @@
     "stackdriver"
   ],
   "scripts": {
-    "cover": "nyc --reporter=lcov mocha build/test/*.js build/test/**/*.js build/test/**/**/*.js && nyc report",
-    "test-no-cover": "mocha build/test/*.js",
-    "test": "npm run cover",
+    "test": "c8 mocha build/test/*.js build/test/**/*.js build/test/**/**/*.js",
     "docs": "jsdoc -c .jsdoc.js",
     "presystem-test": "npm run compile",
-    "system-test": "mocha build/system-test --timeout 600000",
-    "samples-test": "cd samples/ && npm test && cd ../",
+    "system-test": "c8 mocha build/system-test --timeout 600000",
+    "samples-test": "c8 npm run run-samples-tests",
+    "run-samples-tests": "cd samples/ && npm test && cd ../",
```

Comment:
This is a new pattern.  If we want to start doing that, should we consider rolling it out broadly and updating the contributing guide?

Reply:
I instead opted for:

```
cd samples/ && c8 npm test && cd ../
```

The same as the original approach, but with the addition of `c8`.

Reply:
That's great.  I am wondering though - should we consider one hammer PR at this stage to start introducing `c8`?  Maybe cover adding it to all the sample tests at the same time?